.travis.yml

```diff
@@ -30,10 +30,10 @@ jobs:
     # These jobs only occur on tag creation if the prior ones succeed
     - stage: PyPi Deployment
       if: tag IS present
-      python: 3.7.2
+      python: 3.7.5
```

Comment:
Lets not change deployment versions... they are currently working on black magic and I don't want anything breaking with them.

Reply:
WUT ... I though I updated these to 3.7.5 .... *sigh* 
noted i'll ensure its 3.7.5 and py37

Reply:
oh wait im dumb .... shouldn't at least the TOXENV be py37, since the bot no longer supports python3.6 so running the test agaisnt 3.7 would be more realist and find bugs caused by an specific version of python?